melodic/distribution.yaml

```diff
@@ -2299,6 +2299,24 @@ repositories:
       url: https://github.com/ros-controls/urdf_geometry_parser.git
       version: kinetic-devel
     status: developed
+  urdf_tutorial:
+    doc:
+      type: git
+      url: https://github.com/ros/urdf_tutorial.git
+      version: master
+    release:
+      packages:
+      - urdf_sim_tutorial
```

Comment:
Should this package be released from https://github.com/ros/urdf_sim_tutorial instead ?

Reply:
Yeah, lemme fix that.